tnz/zti.py

```diff
@@ -1838,6 +1846,45 @@ def help_plugin(entry=entry):
 
         self.plugins = " ".join(plugins)
 
+    def __register_macros(self):
```

Comment:
I believe the private methods are in alphabetical order. Please move to maintain alphabetical order.